src/sidebar/util/annotations-by-user.ts

```diff
@@ -0,0 +1,42 @@
+import type { APIAnnotationData } from '../../types/api';
```

Comment:
This should go in the `helpers/` directory to preserve the dependency rules (`helper/` can depend on `utils/`, but not vice-versa). In general the `helpers/` directory was intended to contain application-specific functions and types, whereas the `util/` directory is code that is more general and designed to be easy to copy and paste into another app or library as needed.

Reply:
Gotcha! I actually wanted to ask what was the effective difference for those two.